zlib-ng.h.in

```diff
@@ -53,6 +59,9 @@ extern "C" {
 #define ZLIBNG_VER_STATUS 0         /* 0=devel, 1-E=beta, F=Release */
 #define ZLIBNG_VER_MODIFIED 0       /* non-zero if modified externally from zlib-ng */
 
+/* The rest of this file is again uninteresting for the VS resource compiler. */
```

Comment:
Perhaps it would read better if it said something along the lines of, "Only expose version information for the VS resource compiler."